packages/ios-client/Sources/FishjamClient/webrtc/PeerConnectionManager.swift

```diff
@@ -308,6 +273,22 @@ internal class PeerConnectionManager: NSObject, RTCPeerConnectionDelegate {
         return mapping
     }
 
+    private func getTrackIdToBitrates(localTracks: [Track]) -> [String: Int32] {
+        guard let pc = connection else {
+            return [:]
+        }
+
+        var mapping: [String: Int32] = [:]
+        pc.transceivers.forEach { transceiver in
+            guard let trackId: String = transceiver.sender.track?.trackId else {
+                return
+            }
+            mapping[trackId] = 500  // TODO: Change with simulcast
```

Comment:
**suggestion:** more appropriate value would be `1500000` as this value is in bit\s